docker-compose.yml

```diff
@@ -0,0 +1,107 @@
+version: '2.4'
+
+services:
+  postgres:
+    image: gcr.io/seqr-project/postgres:gcloud-dev
```

Comment:
We should build the new images and change the references before deploying

Reply:
done